core/include/jiminy/core/AbstractController.tpp

```diff
@@ -40,11 +40,111 @@ namespace jiminy
 
         if (returnCode == result_t::SUCCESS)
         {
+            // Check in local cache before.
+            auto constantIt = std::find_if(registeredConstants_.begin(),
+                                           registeredConstants_.end(),
+                                           [&fieldName](auto const & element)
+                                           {
+                                               return element.first == fieldName;
+                                           });
+            if (constantIt != registeredConstants_.end())
+            {
+                std::cout << "Error - AbstractController::registerConstant - Constant already registered." << std::endl;
+                return result_t::ERROR_BAD_INPUT;
+            }
             registeredConstants_.emplace_back(fieldName, std::to_string(value));
         }
 
         return returnCode;
     }
+
+    template<>
+    inline result_t AbstractController::registerConstant<char const *>(std::string   const & fieldName,
+                                                                       char  const * const & value)
```

Comment:
Why do you need a special case for ```char const *``` . Can't it be handled  by the templated version as well ?

Reply:
I made a new commit that overloads `to_string` instead of using full template specialization. It is both shorter, clearer, and more flexible.